src/vue/main-menu.vue

```diff
@@ -1,8 +1,21 @@
 <template>
   <div class="alpheios-alignment-app-menu" id="alpheios-main-menu">
       <div class="alpheios-alignment-app-menu__buttons">
-        <button class="alpheios-button-tertiary" id ="alpheios-main-menu-download" @click="$emit('download-data')" >{{ l10n.getMsgS('MAIN_MENU_DOWNLOAD_TITLE') }}</button>
-        <button class="alpheios-button-tertiary" id ="alpheios-main-menu-upload" @click="uploadTexts" >{{ l10n.getMsgS('MAIN_MENU_UPLOAD_TITLE') }}</button>
+        <button class="alpheios-button-tertiary" id ="alpheios-main-menu-add-target" 
+                @click="$emit('add-target')" :disabled="!docSourceEditAvailable" >

```

Comment:
eventually we want to be able to add target texts once the alignment has already started. Ok for this to be in a separate PR but can you make an issue so we don't forget?

Reply:
It is registered here https://github.com/alpheios-project/alignment-editor-new/issues/31

I postponed it to the step, when I would work on edit alignment mode